build.sbt

```diff
@@ -75,3 +75,15 @@ lazy val checkerPlugin = (projectMatrix in file("checker-plugin"))
   .settings(libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test))
   .dependsOn(checkerLibrary)
   .jvmPlatform(scalaVersions = supportedScalaVersions)
+
+lazy val benchmark = (project in file("benchmark"))
+  .settings(name := "serializer-benchmarks")
```

Comment:
Nit: pls keep lazy val name, folder name and project name consistent (identical modulo kebab-case vs camelCase)